lib/matplotlib/backends/web_backend/js/mpl.js

```diff
@@ -474,6 +474,9 @@ mpl.figure.prototype.handle_rubberband = function (fig, msg) {
     var width = Math.abs(x1 - x0);
     var height = Math.abs(y1 - y0);
 
+    fig.rubberband_context.setLineDash([6]);
```

Comment:
Why is this 6 instead of 3 like the others? Also, do we not need to draw white as well, to be consistent with dark background?